plugins/get-fields/pom.xml

```diff
@@ -24,12 +24,10 @@
     <pdi.version>9.1.0.0-SNAPSHOT</pdi.version>
     <version.for.license>${project.version}</version.for.license>
     <js.project.list>requirejs,requirejs-text,angular,require-css,angular-i18n</js.project.list>
-    <angular.version>1.5.8</angular.version>
     <pentaho-osgi-bundles.version>9.1.0.0-SNAPSHOT</pentaho-osgi-bundles.version>
-    <angular-ui-router.version>0.4.2</angular-ui-router.version>
     <requirejs-text.version>2.0.10</requirejs-text.version>
     <require-css.version>0.1.8</require-css.version>
-    <angular-i18n.version>1.5.6</angular-i18n.version>
+    <angular-i18n.version>${angular.version}</angular-i18n.version>
```

Comment:
Could you please use the root pom declarations instead, both w.r.t. properties and dependency management?

Reply:
@dcleao done :)